pandas/core/indexes/base.py

```diff
@@ -3708,7 +3708,7 @@ def _validate_can_reindex(self, indexer: np.ndarray) -> None:
         """
         # trying to reindex on an axis with duplicates
         if not self._index_as_unique and len(indexer):
-            raise ValueError("cannot reindex from a duplicate axis")
+            raise ValueError("Cannot reindex on an axis with duplicate labels")
```

Comment:
I _think_ it's common practice not to use capitalization

SyntaxError: invalid syntax
IndexError: list index out of range
IndentationError: unexpected indent
IndentationError: expected an indented block

Reply:
agree. @aberres can you update & ping on green.